layers/meta-resin-toradex/recipes-support/resin-init/files/resin-init-flasher-board

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+if grep -q "Freescale i.MX6 Quad/DualLite" "/proc/cpuinfo" ; then
+    dd if=/boot/u-boot.imx-dual of=/dev/mmcblk0 conv=notrunc seek=1 bs=1024
+fi
```

Comment:
This assumes you already have a single flashed. Can you do it for single too just for consistency?